### PR TITLE
add groups annotation attribute

### PR DESCRIPTION
**Pull request by Lenny4**
https://github.com/rectorphp/rector/issues/6707

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,27 +21,19 @@ use Rector\Symfony\Set\SymfonySetList;
...
     $parameters->set(
         Option::SYMFONY_CONTAINER_XML_PATH_PARAMETER,
-        __DIR__ . '/var/cache/dev/AppKernelDevDebugContainer.xml'
+        __DIR__ . '/var/cache/dev/App_KernelDevDebugContainer.xml'
     );
+    // endregion
+
+    $containerConfigurator->import(SymfonySetList::SYMFONY_52);
+    $containerConfigurator->import(SymfonySetList::SYMFONY_CODE_QUALITY);
+    $containerConfigurator->import(SymfonySetList::SYMFONY_CONSTRUCTOR_INJECTION);
```
Comment (TomasVotruba) [inline README.md:34]: :+1:

File: README.md [hunk 2]
```diff
@@ -21,27 +21,19 @@ use Rector\Symfony\Set\SymfonySetList;
...
+
+    $containerConfigurator->import(DoctrineSetList::DOCTRINE_ORM_29); // if you use doctrine
```
Comment (TomasVotruba) [inline README.md:36]: This is set from different package. It's separated on purpose, to keep domain as isolated as possible.

File: config/sets/symfony/symfony52-validator-attributes.php [hunk 3]
```diff
@@ -258,6 +258,10 @@
                     'Symfony\Component\Validator\Constraints\Valid',
                     'Symfony\Component\Validator\Constraints\Valid'
                 ),
+                new AnnotationToAttribute(
+                    'Symfony\Component\Serializer\Annotation\Groups',
+                    'Symfony\Component\Serializer\Annotation\Groups'
+                ),
```
Comment (TomasVotruba) [inline config/sets/symfony/symfony52-validator-attributes.php:264]: :+1:

File: README.md (hunk 2 above)
Comment (Lenny4) [reply README.md]: @TomasVotruba done

**Review by samsonasik — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**